esrally/driver/runner.py

```diff
@@ -775,7 +775,11 @@ class Query(Runner):
     * `pages`: Number of pages to retrieve at most for this scroll. If a scroll query does yield less results than the specified number of
                pages we will terminate earlier.
     * `results-per-page`: Number of results to retrieve per page.
+    * `request-id`: a user specified string identifying the query. This will be included in rally metrics to allow query
+                execution to be tracked.
+    
```

Comment:
@gingerwizard as mentioned by the lint failure in CI, this newline needs to go away.